openslides/motions/static/js/motions/site.js

```diff
@@ -1152,6 +1230,23 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
         /// Prefix for the identifier for amendments
         gettext('A');
     }
-]);
+])
+
+//filter selfid from input.id
+//usage |notself: selfid
+.filter('notself', function() {
```

Comment:
Please move this filter to core/site.js - place it near other filter definitions. Remove this  filter from specific apps.